casper/src/main/scala/coop/rchain/casper/genesis/contracts/RevGenerator.scala

```diff
@@ -24,8 +29,8 @@ final case class RevGenerator(genesisAddress: RevAddress, userVaults: Seq[Vault]
        #         *genesisVaultCh
        #       )
        #       | for (@(true, genesisVault) <- genesisVaultCh) {
-       #         new genesisAuthKeyCh in {
-       #           @RevVault!("deployerAuthKey", *genesisAuthKeyCh)
+       #         new genesisAuthKeyCh, deployerId(`rho:rchain:deployerId`) in {
```

Comment:
this too should be migrated to an unf-name based genesis vault (the only one having rev supply upon its creation). So there's no need for genesisPk in RevGenerator. Not until we have the unf-name based vaults though. Can you link this comment to the relevant jira issue?

Reply:
Done: https://rchain.atlassian.net/browse/RCHAIN-3617